orchagent/tunneldecaporch.cpp

```diff
@@ -156,15 +169,47 @@ void TunnelDecapOrch::doTask(Consumer& consumer)
                     }
                     if (exists)
                     {
-                        setTunnelAttribute(fvField(i), ttl_mode, tunnelTable.find(key)->second.tunnel_id);
+                        setTunnelAttribute(fvField(i), ttl_mode, tunnel_id);
+                    }
+                }
+                else if (fvField(i) == DECAP_DSCP_TO_DC_MAP)
+                {
+                    string dscp_to_dc_map_name = fvValue(i);
+                    // TODO: Validate DSCP_TO_TC_MAP map name, and get map id
+                    if (exists)
+                    {
+                        setTunnelAttribute(fvField(i), dscp_to_dc_map, tunnel_id);
                     }
                 }
+                else if (fvField(i) == DECAP_TC_TO_PG_MAP)
+                {
+                    string tc_to_pg_map_name = fvValue(i);
+                    // TODO: Validate TC_TO_PG_MAP name, and get map id
+                    if (exists)
+                    {
+                        setTunnelAttribute(fvField(i), tc_to_pg_map, tunnel_id);
+                    }
+                }
+                else if (fvField(i) == ENCAP_TC_TO_DSCP_MAP)
```

Comment:
Lets remove the ENCAP fields from this file as it is only handling decap tunnel here

Reply:
The reason for I put it here is to parse entry from config db and store the map id into `TunnelTable`. The map will be used in `muxorch`. So I added a public inerface ` getTCToDSCPMap` and `getTCToQueueMap`.